src/utility/hexagons.js

```diff
@@ -685,6 +672,25 @@ var HexGrid = class HexGrid {
 				game.UI.showCreature(game.activeCreature.type, game.activeCreature.player.id);
 			}
 		};
+		
+		let onCreatureHover = (creature, queueEffect) => {
+			if (creature.type == "--"){
+				if (creature === game.activeCreature){
```

Comment:
For consistency, there's usually space in between `) {` and comments are capitalized :)